runtime/include/j9cfg.h.ftl

```diff
@@ -33,6 +33,8 @@ extern "C" {
 #endif
 
 #include "omrcfg.h"
+#include "../util/openj9_version_info.h"
```

Comment:
Why is this header file in util/ and not in either /include or /oti?

Reply:
We typically change the include path rather than using relative paths to get to header files.

I think moving the header is probably a better choice.

Reply:
Moved generated `openj9_version_info.h` from folder `util` to `include`.